libical will fail pkgconf test pipeline

If the pkgconf test pipeline is added to libical it will fail in the following way:

```
2024/11/12 23:12:02 INFO running step "test/pkgconf"
2024/11/12 23:12:02 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:12:02 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:12:02 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:12:02 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:12:02 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:02 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:02 WARN + basename /home/build/melange-out/libical-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:02 WARN + dev_pkg=libical-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:02 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:02 WARN + apk info -L libical-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:02 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:02 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:02 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:02 WARN + grep -q ^Name: usr/lib/pkgconfig/libical-glib.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:02 WARN + basename usr/lib/pkgconfig/libical-glib.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:02 WARN + lib_name=libical-glib uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:02 WARN + echo usr/lib/pkgconfig/libical-glib.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:02 WARN + grep -q '^usr/lib/pkgconfig/libical-glib.pc$\|^usr/share/pkgconfig/libical-glib.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:02 WARN + pkgconf --exists libical-glib uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:02 WARN + pkgconf --print-errors --exists libical-glib uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:02 WARN Package glib-2.0 was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:02 WARN Perhaps you should add the directory containing `glib-2.0.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:02 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:02 WARN Package 'glib-2.0', required by 'libical-glib', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:02 WARN Package 'gobject-2.0', required by 'libical-glib', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:04 INFO pod 73e2bd760efb29e23931cd4167172d4e7f32f6a5da5efee3273e2e69f3bc2e46 terminated
2024/11/12 23:12:04 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:12:04 INFO   workspace dir: /home/user/tmp/melange-workspace-3126792694
2024/11/12 23:12:04 INFO   guest dir: /home/user/tmp/melange-guest-3110697893
2024/11/12 23:12:04 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/libical] Error 1
```

--- a/libical.yaml
+++ b/libical.yaml
@@ -2,7 +2,7 @@
 package:
   name: libical
   version: 3.0.16
-  epoch: 4
+  epoch: 5
   description: Reference implementation of the iCalendar format
   copyright:
     - license: LGPL-2.1-only OR MPL-2.0
@@ -51,6 +51,13 @@ subpackages:
     pipeline:
       - uses: split/dev
     description: libical dev
+    dependencies:
+      runtime:
+        - glib-dev
+        - gobject-introspection-dev
+    test:
+      pipeline:
+        - uses: test/pkgconf
 
 update:
   release-monitor:

